_targets.R

```diff
@@ -7,14 +7,26 @@ options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
 p1_targets_list <- list(
-  tar_target(
-    site_data,
-    download_nwis_data(),
+  tar_target(site_data_01427207, download_nwis_site_data('01427207')),
+  tar_target(site_data_01432160, download_nwis_site_data('01432160')),
+  tar_target(site_data_01436690, download_nwis_site_data('01436690')),
+  tar_target(site_data_01466500, download_nwis_site_data('01466500')),
+  
+  tar_target(site_data,
```

Comment:
Since a CSV is the result, something like `site_data_csv` would be great. Using this pattern (adding `_[file extension]` to the end of a file target helps make it extra clear (and reminds you when it is used as input to other targets) that it is a filepath, not the loaded data.